src/Tools/dotnet-monitor/Egress/Configuration/EgressProviderConfigureNamedOptions.cs

```diff
@@ -34,7 +34,23 @@ public void Configure(string name, TOptions options)
                 IConfigurationSection providerOptionsSection = providerTypeSection.GetSection(name);
                 if (providerOptionsSection.Exists())
                 {
-                    providerOptionsSection.Bind(options);
+                    if (options is ExtensionEgressProviderOptions extensionOptions)
+                    {
+                        var children = providerOptionsSection.GetChildren();
```

Comment:
Why is this needed? Are you working around this issue? https://github.com/dotnet/runtime/issues/77246